src/main/java/edu/harvard/iq/dataverse/License.java

```diff
@@ -0,0 +1,141 @@
+package edu.harvard.iq.dataverse;
+
+import java.util.Objects;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.NamedQueries;
+import javax.persistence.NamedQuery;
+import javax.persistence.Table;
+import javax.persistence.UniqueConstraint;
+
+/**
+ * @author Jing Ma
+ */
+ @NamedQueries({
+    @NamedQuery( name="License.findAll",
+                query="SELECT l FROM License l"),
+    @NamedQuery( name="License.findById",
+            query = "SELECT l FROM License l WHERE l.id=:id"),
+    @NamedQuery( name="License.deleteById",
+                query="DELETE FROM License l WHERE l.id=:id")
+
+})
+@Entity
+@Table(uniqueConstraints = {
+      @UniqueConstraint(columnNames = "name"),
+      @UniqueConstraint(columnNames = "uri")}
+)
+public class License {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Column(columnDefinition="TEXT", nullable = false)
+    private String name;
+
+    @Column(columnDefinition="TEXT")
+    private String shortDescription;
+
+    @Column(columnDefinition="TEXT", nullable = false)
+    private String uri;
+
+    @Column(columnDefinition="TEXT")
+    private String iconUrl;
```

Comment:
Would it be better if we use the URL class here, we don't want any string here actually

Reply:
I agree